app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -96,9 +97,6 @@
     
     @UiElement(R.id.screen_login_banner_pane)
     private View banner;
-    
-    @UiElement(R.id.str_version)
-    private TextView versionDisplay;

```

Comment:
This info can be accessed via the login screen by selecting the "About CommCare" menu entry. Pretty sure this is just debris from the old ui.